### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by johanmalm**
@Consolatis @stefonarch Appreciate comments before I merge.

### CODE REVIEW

File: src/faq.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Frequently Asked Questions
+
+## Unset keybinds
+
+### Q: How do I get `Alt+Left/Right` to not move windows? I want to these to navigate forward/backwards in Nautilus and Firefox.
```
Comment (Consolatis) [inline src/faq.md:5]: `s/I want to these to/I want these to/`

File: src/faq.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+You could either copy `rc.xml.all` and just remove the entries you do not want, such as `<keybind key="A-Left">`
+
+Or you could use the `None` action, which is simpler and allows you to simply do this:
+
+```
+  <keyboard>
+    <default/>
+    <keybind key="A-Left" name.action="None"/>
+    <keybind key="A-Right" name.action="None"/>
+  </keyboard>
```
Comment (Consolatis) [inline src/faq.md:16]: Even easier would be
```xml
  <keyboard>
    <default/>
    <keybind key="A-Left" />
    <keybind key="A-Right" />
  </keyboard>
```

File: src/faq.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+> Is it possible to launch wofi with only the Super key?
+
+No. I'm afraid that is not yet implemented.
+
+### Q: Is it possible to remove gaps and rounded corners for maximized windows?
+
+Almost. `<action name="SnapToEdge" direction="center" />` respects gaps. This
+action resizes the window to the full size of the output (minus space for
+layer-shell clients and gaps), but it doesn't formally 'maximize' it.
```
Comment (Consolatis) [inline src/faq.md:39]: I don't understand this solution,  Wouldn't it still show gaps and rounded corners?

**Comment by Consolatis** [comment]
Besides the comments above, LGTM.

File: src/faq.md (hunk 1 above)
Comment (johanmalm) [reply src/faq.md:5]: Done.

File: src/faq.md (hunk 2 above)
Comment (johanmalm) [reply src/faq.md:16]: More elegant. Thanks. Done.

File: src/faq.md (hunk 3 above)
Comment (johanmalm) [reply src/faq.md:39]: Oh, I misread the question. I thought they were asking to *have* gaps in maximized state.

**Comment by stefonarch** [comment]
> @Consolatis @stefonarch Appreciate comments before I merge.

Somehow I missed that, sorry. I would had replaced "Nautilus" with "File managers", at least pcmanfm-qt uses those too.

**Comment by johanmalm** [comment]
> I would had replaced "Nautilus" with "File managers", at least pcmanfm-qt uses those too.

Done

**Pull request merged**